From 046ed22b9659a3a6f23747954ed861ebdb5a9cc6 Mon Sep 17 00:00:00 2001
From: Yann Gautier <yann.gautier@foss.st.com>
Date: Wed, 17 Jan 2024 17:55:35 +0100
Subject: [PATCH 479/954] fix(stm32mp2): correct NAND error message

Update error message when accessing NAND2_OTP.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/arm-trusted-firmware.git ea67742b4f0b08972b40990433cc12be52e07f6a fix(stm32mp2): correct NAND error message]
Change-Id: I90b5b4d5b833dac3092af6d86d7706d165d7cf1b
Signed-off-by: Yann Gautier <yann.gautier@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/tf-a/+/352185
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
Reviewed-by: Maxime MERE <maxime.mere@foss.st.com>
Domain-Review: Christophe KERELLO <christophe.kerello@st.com>
---
 plat/st/stm32mp2/stm32mp2_boot_device.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/plat/st/stm32mp2/stm32mp2_boot_device.c b/plat/st/stm32mp2/stm32mp2_boot_device.c
index a391f6498e..2fd275c262 100644
--- a/plat/st/stm32mp2/stm32mp2_boot_device.c
+++ b/plat/st/stm32mp2/stm32mp2_boot_device.c
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2021-2022, STMicroelectronics - All Rights Reserved
+ * Copyright (c) 2021-2024, STMicroelectronics - All Rights Reserved
  *
  * SPDX-License-Identifier: BSD-3-Clause
  */
@@ -36,7 +36,7 @@ static int get_data_from_otp(struct nand_device *nand_dev, bool is_slc)
 	}
 
 	if (stm32_get_otp_value(NAND2_OTP, &nand2_param) != 0) {
-		ERROR("BSEC: NAND_OTP Error\n");
+		ERROR("BSEC: NAND2_OTP Error\n");
 		return -EACCES;
 	}
 
-- 
2.53.0

